src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Duke class.
+ * Uses storage class to load existing data.
+ * If there is not existing file, it will create one.
+ * It takes in user input and create the task object.
+ * It will save the final list into the file when programmes end.
+ */
+public class Duke {
+    static void drawLine() {
+        System.out.println("___________________________________________");
+    }
+
+    public static void main(String[] args) throws IOException {
```

Comment:
Should an IOException be allowed to crash the whole program? It could be better to handle it in the functions that could cause the exception.